### DISCUSSION TITLE
Wind speed

### DISCUSSION THREAD

**Discussion by MarkWildig** [cgiesche/streamdeck-homeassistant#246] [Help and Support] [answered]
Hi When displaying wind speed from a tem[est weather station the stream deck reports some amazing long 8 digit number, i think its just at too high number of digits as the first number looks like the speed in MPH 
Thank you so much 
Mark

**Comment by cgiesche** [comment, marked as answer]
Hi Mark,

You can use the following Nunjucks expression in the custom-labels field to truncate the excess digits: `{{ state | round(2, 'floor') }}`. (Replace `state` by another attribute name, if necessary).
